corehq/apps/reports/models.py

```diff
@@ -268,7 +269,11 @@ def _dispatcher(self):
             if dispatcher.prefix == self.report_type:
                 return dispatcher()
 
-        raise Exception("Unknown dispatcher: %s" % self.report_type)
+        logging.error("This saved-report (id: %s) is unknown (report_type: %s). (Might be a legacy report)" % (
```

Comment:
I don't think `logging.error` is a good way for us to track these. if we care about the error we should probably use `soft_assert` or `notify_error`. if we don't we should just swallow silently. in this case I think you're right that we do care, so good to put it somewhere.

where is the exception caught? i'm thinking it might be more appropriate to add the logging there since this code is already reraising it. that said, if it's being caught in a very general way then it makes sense to keep here.